README.md

```diff
@@ -21,6 +21,16 @@ NMState is aimed to satisfy enterprise needs to manage host networking through
 a northbound declarative API and multi provider support on the southbound.
 NetworkManager acts as the main (and currently the only) provider supported.
 
+## Contact
+nmstate uses the Network [Manager mailing
```

Comment:
... 'the [NetworkManager mainling' ...

You could also use '*' around nmstate name.

Reply:
changed